Notification.Api/Contracts/SmsAlertContract.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Notification.Api.Models
+{
+    public class SmsAlertContract
+    {
+        public string ContactName { get; set; }
+        public string ContactStatus { get; set; }
```

Comment:
I would make this more generic, having just the UserPhoneNumber and a "Message" field, then we can use this for all of our notifications.